src/main/java/com/ioet/bpm/bpmorganizationsapi/repositories/OrganizationRepository.java

```diff
@@ -7,13 +7,16 @@
 import org.springframework.data.repository.CrudRepository;
 
 import java.util.List;
+import java.util.Optional;
 
 
 @EnableScan
 public interface OrganizationRepository extends CrudRepository <Organization, String> {
 
-    public List<Organization> findAll();
+    public List<Organization> findAllByUid();
+
+    public  Optional<Organization> findAllByUid(String uid);
+
 
```

Comment:
What the heck is doing this service?

Reply:
I will delete it. Actually I dont know why I wrote that.